bin/generate-page

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env php
+<?php declare(strict_types=1);
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+require __DIR__ . '/../vendor/codeigniter4/codeigniter4/system/Test/bootstrap.php';
+
+use CodeIgniter\CLI\CLI;
```

Comment:
Since we are coupling to the framework... is there any sense to making this into an actual CI4 Command? If we did: could we still launch it via `spark` if the framework is embedded in **vendor/**?

Reply:
I'm not sure 🤔 It looks like we would have to add this command to CI4, and I'm not a big fan of having it in the core.

Reply:
I was thinking the same for converting the scripts in `bin/` to actual `spark` commands here in the repo. For the meantime, this should be fine. We can convert later on.